src/signals/incident-management/containers/DefaultTextsAdmin/index.js

```diff
@@ -44,12 +45,14 @@ const DefaultTextsAdmin = ({
   },
 }) => (
   <Fragment>
-    <StyledPageHeader title="Beheer standaard teksten" />
     <Row>
+      <Column span={12}>
+        <StyledH1 $as="h1">Beheer standaard teksten</StyledH1>
```

Comment:
Isn't the default element an `<h1>` already?

Reply:
yes gedaan